ipn/desktop/sessions_windows.go

```diff
@@ -670,3 +670,38 @@ func (cs _WTS_CONNECTSTATE_CLASS) ToSessionStatus() SessionStatus {
 		return ClosedSession
 	}
 }
+
+var (
+	procGetWindowLongPtrW *windows.LazyProc
+	procSetWindowLongPtrW *windows.LazyProc
+)
+
+func init() {
+	// GetWindowLongPtrW and SetWindowLongPtrW are only available on 64-bit platforms.
+	// https://web.archive.org/web/20250414195520/https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowlongptrw
+	if is64bit := unsafe.Sizeof(uintptr(0)) == 8; is64bit {
```

Comment:
Please change this to use `runtime.GOARCH == "386"` as the check to make it consistent with the rest of the codebase.